tests/RobotTest.php

```diff
@@ -29,6 +29,22 @@ public function testBasicDisallow()
     }
 
     /**
+     * Test that we are not allowed to access a resource which is forbidden using the isDisallowed method
+     */
+    public function testBasicDisallow_Disallowed_method()
```

Comment:
I think you could replace this test case with an extra assertion in ```testBasicDisallow``` so it checks isDisallowed as well as isAllowed

ditto with ```testBasicIsDisallowed_returning_false``` - its assertion could go into ```testBasicAllow```